webui/react/src/styles/antd.scss

```diff
@@ -1,7 +1,51 @@
-.ant-select > .ant-select-arrow {
-  transform: rotate(0deg);
-  transition: transform 0.2s linear;
-}
-.ant-select.ant-select-open > .ant-select-arrow {
-  transform: rotate(180deg);
+html {
+  /* Button */
+
+  .ant-btn[disabled] {
+    background-color: var(--theme-colors-monochrome-11);
+    border-color: var(--theme-colors-monochrome-11);
+    color: var(--theme-colors-monochrome-17);
+  }
+
+  /* Form */
+
+  .ant-form-item-explain {
```

Comment:
are these for the `determinedAuth` changes?

Reply:
they're changes for general antd form errors